DESIGN.md

```diff
@@ -0,0 +1,235 @@
+[-](url) Start Date: 2022-06-09
+- TTL: June 15th, 2022
+- Champion: Huy Nguyen
+- Main reviewer: Tao Liu
+- Owner team: OpenSearch
+- Stakeholders: GitHub Community
+- OpenSearch Issue: #8 
+
+# Executive Summary
+
+
+## Problem Statement
+
+While OpenSearch (abbreviated as **OS**), an open-source search and analytics
+suite, and its corresponding visualization tool OpenSearch Dashboards 
+(abbreviated as **OSD**) are powerful open-source tools, there has been a need 
+by the community test out its features. Currently, the most commonly used method 
+of demoing OS and OSD is by downloading their Docker images and running them 
+on the user's local machine. 
+
+![sample](https://user-images.githubusercontent.com/73027756/173409363-875057fe-6a2d-4b95-85cc-c8c3831d08c7.png)
+
+While serviceable for some people, it was clear that a more 
+convenient method was necessary to spread OS and OSD's capabilities to a wider 
+customer audience.
+
+
+## Goals
+
+To make OS and OSD more convenient for our existing and prospective customers,
+there were several key features that this OS/OSD application must implement:
+
+- **Secure:** the application must be secure for users to gain trust in not 
+    only the application but also OS/OSD. SSL certificates must be made and
+    used.
+- **Easily Accessible:** the application should not require the user to 
+    download, sign-up, or any other extraneous step to play around with 
+    OS/OSD's features.
+- **Scaleable:** because this application showcases the capabilities of 
+    OS/OSD, this application will be a first impression for many customers.
+    Thus, the application should service them in a fast, secure, and reliable
+    manner.
+
+
+## Proposal
+
+To address this need, a demo website we call Playground was proposed that 
```

Comment:
Could you please normalize the name which comments in the issue? https://github.com/opensearch-project/dashboards-anywhere/issues/9#issuecomment-1154556646

Reply:
Done, the demo site is now named as "OpenSearch Playground."

Reply:
I also added some more information taken from the team review.

Reply:
This is design based on short term, can we add some information for the title and design content. thanks

Reply:
Could you please change the file to PLAYGROUND_SHORT_TERM_DESIGN.md ? we will add PLAYGROUND_LONG_TERM_DESIGN.md later on.

Reply:
Thanks. All looks good to me.